text.go

```diff
@@ -557,9 +550,9 @@ func (t *Text) RenderDecoration(r Renderer, m Matrix) {
 	for _, line := range t.lines {
 		for _, deco := range line.decos {
 			p := deco.face.Decorate(deco.x1 - deco.x0)
-			p = p.Transform(Identity.Mul(m).Translate(deco.x0, line.y+deco.face.Voffset))
+			p = p.Translate(deco.x0, line.y+deco.face.Voffset)
 			style.FillColor = deco.face.Color
-			r.RenderPath(p, style, Identity)
+			r.RenderPath(p, style, m)
```

Comment:
I can take a stab at this tomorrow, but does this perhaps invert the order of transformation matrix multiplication? It used to be `m * translation`, does it now end up being `translation * m` (which gives different results)?

Reply:
`r.RenderPath` usually calls `p.Transform(m)`.

Previously it meant:

`pRenderDecoration = Decorate.Transform(Identity.Mul(m).Translate(deco...))`
And calling RenderPath applies `pRenderDecoration.Transform(Identity)`

Now it will be

`pRenderDecoration = Decorate.Translate(deco....)`
And calling RenderPath applies `pRenderDecoration.Transform(m)`

**But `p.Transform(m)` is actually doing the multiplication in the other order: `m.p` (m being a matrix, and p the path).**

So the old one ends being:
```
Pfinal = (Identity * ((Mm * Mtranslate) * Pdecorate)
```
And the new one
```
Pfinal = Mm * (Mtranslate * Pdecorate)
```

So this is actually the same (since matrix multiplication is associative)